libp2p-comit/Cargo.toml

```diff
@@ -8,9 +8,9 @@ description = "Implementation of the COMIT messaging protocol"
 [dependencies]
 bytes = "0.5"
 derivative = "2"
-futures = { version = "0.3", features = ["alloc", "io-compat"] }
+futures = { version = "0.3", default-features = false }
```

Comment:
Removing the need for `io-compat` completely removes `futures 0.1.29` from our dependency tree :tada: